TestModels/aws-sdks/ddbv2/test/TestDDBv2.dfy

```diff
@@ -0,0 +1,206 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+include "../src/Index.dfy"
+
+module TestDDBv2 {
+    import DDB = Com.Amazonaws.Dynamodb
+    import opened StandardLibrary.UInt
+    import opened Wrappers
+    
+    const tableNameTest : DDB.Types.TableName := "TestTable";
+    const secIndex : DDB.Types.IndexName := "Active-Keys"
+
+    // We should have basic tests for what the MPL is using DDB for.
+    // MPL will interact with DDB through GET, PUT, and Query
+
+    // Queries are used in the Hierarchy Keyring for OnEncrypt
+    method {:test} BasicQueryTests() {
+        var attributeNameMap := map[
+          "#bkid"   := "branch-key-id",
+          "#status" := "status"
+        ];
+
+        var attributeValueMap: DDB.Types.AttributeMap := map[
+          ":bkid"   := DDB.Types.AttributeValue.S("aws-kms-h"),
+          ":status" := DDB.Types.AttributeValue.S("ACTIVE")
+        ];
+
+        var queryInput := DDB.Types.QueryInput(
+            TableName := tableNameTest,
+            IndexName := DDB.Wrappers.Some(secIndex),
+            Select := DDB.Wrappers.None,
+            AttributesToGet := DDB.Wrappers.None,
+            Limit := DDB.Wrappers.None,
+            ConsistentRead :=  DDB.Wrappers.None,
+            KeyConditions := DDB.Wrappers.None,
+            QueryFilter := DDB.Wrappers.None,
+            ConditionalOperator := DDB.Wrappers.None,
+            ScanIndexForward := DDB.Wrappers.None,
+            ExclusiveStartKey := DDB.Wrappers.None,
+            ReturnConsumedCapacity :=  DDB.Wrappers.None,
+            ProjectionExpression := DDB.Wrappers.None,
+            FilterExpression := DDB.Wrappers.None,
+            KeyConditionExpression := DDB.Wrappers.Some(
+                "#status = :status and #bkid = :bkid"
+            ),
+            ExpressionAttributeNames := DDB.Wrappers.Some(attributeNameMap),
+            ExpressionAttributeValues := DDB.Wrappers.Some(attributeValueMap)
+        );
+        BasicQueryTest(input := queryInput);
+    }
+
+    // MPL uses Get in the Hierarchy Keyring for OnDecrypt
+    method {:test} BasicGetTests() {
+        var Key2Get: DDB.Types.Key := map[
+          "branch-key-id" := DDB.Types.AttributeValue.S("aws-kms-h"),
+          "version" := DDB.Types.AttributeValue.S("1")
+        ];
+        var getInput := DDB.Types.GetItemInput(
+            TableName := tableNameTest,
+            Key := Key2Get,
+            AttributesToGet := DDB.Wrappers.None,
```

Comment:
Just FYI we don't need all of these any more, because the codegen sets a default value now (I know this is just copied from the existing ddb test model)

Reply:
The `Wrappers.None` part? Wasn't it always the case that it was `None` if no value was supplied? I though we explicity added the `Wrappers.None` to just be verbose in the tests.